src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,49 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<K, V> {
+    private K first;
+    private V second;
 
+    private Pair(K first, V second) {
+        this.first = first;
+        this.second = second;
+    }
+
+    public static <K, V> Pair of(K argument1, V argument2) {
+        return new Pair(argument1, argument2);
```

Comment:
```suggestion
    public static <K, V> Pair<K, V> of(K argument1, V argument2) {
        return new Pair<>(argument1, argument2);
```
let's parametrize the returns type too

Reply:
done